Apex Language/APEX_2/OppOwnermail.cls

```diff
@@ -0,0 +1,10 @@
+global class OppOwnermail implements Schedulable {
+    global void execute(SchedulableContext ctx) {
+        AggregateResult[] groupedResults = [SELECT owner.email email FROM Opportunity WHERE LastModifiedDate <  LAST_N_DAYS:30 GROUP BY owner.email];
```

Comment:
why `GROUP BY`?

Reply:
here I use Group by for grouping same type email Id